ibm_catalog.json

```diff
@@ -19,25 +18,41 @@
         "red_hat_openshift"
       ],
       "short_description": "Automates Red Hat OpenShift VPC cluster deployment on IBM Cloud, offering full configurability and flexibility for diverse workloads.",
-      "long_description": "This deployable architecture deploys a Red Hat OpenShift cluster on IBM Cloud. This is an experimental tile and not suitable for production workloads. Stay here if you want to try an experimental version with the [Optional and swappable components](https://cloud.ibm.com/docs/secure-enterprise?topic=secure-enterprise-choose-plan-process#optional-swappable) capability.",
+      "long_description": "The Red Hat OpenShift Container Platform on VPC provides the tools to deploy a Red Hat OpenShift Container Platform cluster in a single Virtual Private Cloud (VPC) network. The VPC is a multi-zoned, multi-subnet implementation that keeps your VPC secure and highly available.\n",
       "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/blob/main/README.md",
       "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/main/images/ocp_icon.svg",
       "provider_name": "IBM",
       "features": [
         {
-          "title": "OCP on IBM Cloud",
+          "title": "Red Hat OpenShift",
           "description": "Create and configures a Red Hat OpenShift VPC cluster on IBM Cloud."
         },
         {
-          "title": "Object Storage bucket for OpenShift internal registry storage.",
+          "title": "Creates worker pools",
+          "description": "With worker pools, you can group and manage worker nodes with similar configurations, such as compute resources and availability zones."
+        },
+        {
+          "title": "Configures subnets for containers",
+          "description": "Configures the subnets for the cluster, and specifies the subnets to deploy the worker nodes in."
+        },
+        {
+          "title": "Supports private and public endpoints",
+          "description": "Configures private and public endpoints for the cluster."
+        },
+        {
+          "title": "Configures ingress",
+          "description": "Configures the ingress controller for the cluster, responsible for routing external traffic to the appropriate services within the cluster."
+        },
+        {
+          "title": "Object Storage bucket for OpenShift internal registry storage",
```

Comment:
Please provide links as discussed for all the features. 
Title can be without create/configures/supports etc., add this info in description.

Reply:
Refer here - https://github.ibm.com/GoldenEye/issues/issues/13592#issuecomment-112085880